crates/prqlc/Cargo.toml

```diff
@@ -5,11 +5,9 @@ name = "prqlc"
 edition.workspace = true
 license.workspace = true
 repository.workspace = true
-rust-version.workspace = true
+rust-version = "1.70.0"
 version.workspace = true
 
-metadata.msrv = "1.65.0"
```

Comment:
I think this changes the wrong thing:
- The rust version is the standard version when developing etc. There's not much reason to have that be anything but latest / latest-1. We definitely want it the same throughout the workspace
- The MSRV is the version we guarantee we can still compile against

Because compilers are always backward-compatible, it's always fine to use a more recent version of the compiler if it's available

Reply:
> The rust version is the standard version when developing etc. There's not much reason to have that be anything but latest / latest-1. We definitely want it the same throughout the workspace

Isn't that `rust-toolchain`?
https://github.com/PRQL/prql/blob/33b71715136380255482e902c8afe92bd5a8fe10/rust-toolchain.toml#L1-L4

<https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field> says:

> The rust-version field is an optional key that tells cargo what version of the Rust language and compiler your package can be compiled with. If the currently selected version of the Rust compiler is older than the stated version, cargo will exit with an error, telling the user what version is required.

Reply:
I think the reason we're using `metadata.msrv` instead of `rust-version` here is because cargo-msrv doesn't support inheritance from the workspace. foresterre/cargo-msrv#590

Reply:
Sorry sorry, you're completely correct...